pkg/mcs/resource_manager/server/manager.go

```diff
@@ -109,8 +112,8 @@ func (m *Manager) DeleteResourceGroup(name string) error {
 	return nil
 }
 
-// GetResourceGroup returns a copy of a resource group.
-func (m *Manager) GetResourceGroup(name string) *ResourceGroup {
+// GetResourceDuplicateGroup returns a copy of a resource group.
+func (m *Manager) GetResourceDuplicateGroup(name string) *ResourceGroup {
```

Comment:
`GetResourceDuplicateGroup` sounds like it will get all duplicated groups, which is confusing.

```suggestion
func (m *Manager) GetResourceGroupCopy(name string) *ResourceGroup {
```